README.md

```diff
@@ -28,7 +28,7 @@ Users hosting documentation on Read The Docs *do not* need to set any of the fol
 * `ogp_site_name`
     * This is not required. Name of the site. This is displayed above the title.
 * `ogp_image`
-    * This is not required. Link to image to show.
+    * This is not required. Link to image to show. Note that all relative paths are converted to be relative to the root of the html output.
```

Comment:
We should add a comment to see an example below.

Reply:
what do you mean? there are no examples of the relative paths, I could just add another line with a example to the same sentence?